deploy/src/native_to_erc/foreign.js

```diff
@@ -37,7 +41,7 @@ async function deployForeign() {
   console.log('========================================\n')
 
   console.log('\n[Foreign] deploying POA20 token')
```

Comment:
Replace `POA20` in the log by the corresponding value of `BRIDGEABLE_TOKEN_SYMBOL` variable.
Please do this for all places where it is applicable.

Reply:
will do